text/0000-partial-template.md

```diff
@@ -72,10 +68,24 @@ Why should we *not* do this? Please consider:
 
 ## Alternatives
 
-The other design choice would be to compile time import via an include syntax.
+Partials imported via the `template` tag and `lwc:partial` attribute.
+
+- Introduce `lwc:partial="./file.html"` for compiled time content.
+- Scoped `{variables}` within templates passed via attributes.
+- Question: Would `lwc:partial={templateUrl}` not work? What about `lwc:partial={template}` via
+  an `import`.
 
 ```
-<template lwc:include="./template.html"></template>
+// foo.html
+<template><p>Value: {value}</p></template>

```

Comment:
there are more details missing here, what you can pass into a template tag are attributes, what you can use in `{something}` are attributes, basically:

```
<template lwc:partial="./foo" some-value={myBeforeValue}></template>
```

In which case, you will use:

```
<template><p>Value: {someValue}</p></template>
```